helpers/truncate.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+/* функция, которая обрезает текстовое содержимое
+если оно превышает заданное число символов */
+
+function truncate($description, $length = 300)
```

Comment:
Нужно добавить типизацию
```suggestion
function truncate(string $description, int $length = 300): string
```